srml/democracy/src/lib.rs

```diff
@@ -417,6 +512,11 @@ impl<T: Trait> Module<T> {
 	}
 
 	/// Remove all info on a referendum.
+	///
+	/// # <weight>
+	/// - **FLAG**
+	/// - Depends on size of `voters_for` array, is it possible to grow this array?
```

Comment:
`VotersFor` is a `vec`, not an array. It is possible for this to grow.
```suggestion
	/// - Depends on size of `VotersFor` vector.
```

Reply:
@lsaether I think this was a self-note from you so won't touch it.

Reply:
Thanks, this suggestion can be merge. I've noted the same on `tally` which is where it would be higher concern.